.github/workflows/build-nomad-spark-distro.yaml

```diff
@@ -13,6 +13,15 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: '1.8'
+      - name: Install Python 3.8
```

Comment:
Do we want to install `python 3.8` in the build CI file too ?
https://github.com/ExpediaGroup/nomad-spark/blob/nomad-spark-3.0.1/.github/workflows/mvn-branch-ci.yaml#L45

Reply:
Yes, though I will address that as part of the DATACO-988 ticket. Was considering making that change here, but this PR is restricted as I will be modifying `mvn-branch-ci.yaml` a lot for 988 anyways.